Update README.md #29

catclops opened the pull request:
Please review and let me know if I need to add or remove anything.

catclops committed:
Create server.js

catclops committed:
Update README.md

Updated the README.md to give brief background info on the project and to link to our tutorial along with the script.

catclops committed:
Delete server.js

catclops committed:
Update README.md

catclops committed:
Update README.md

catclops committed:
Updates README.md #29

catclops committed:
Update README.md

tacostreets reviewed (changes requested):
Hey Vu! This looks really great. One thing I think would be good to change is the "Express Installation" headline. It might be confusing since ours is a MEAN stack and includes "ExpressJS". So something like "Quick Install" or "Abridged Install" or something more clever you come up with. ;-)

tacostreets reviewed (changes requested):
One more thing. Please include in the Background section what is in a MEAN stack.

catclops committed:
Update README.md

Added your review notes such as a brief description of MEAN and changed the express install title.

tacostreets commented:
Looks good! Merging.
